content/influxdb/v2.0/tools/clis.md

```diff
@@ -0,0 +1,28 @@
+---
+title: InfluxDB CLIs
```

Comment:
How about "Use InfluxDB CLIs" so it feels more procedural than reference?

Then, we can add procedures/examples for how to "Write and query data", "Generate InfluxDB templates", "Export data", "Manage organizations" moving forward

Reply:
Updated. The nav now looks like this:

![image](https://user-images.githubusercontent.com/181484/92973492-8b93eb00-f441-11ea-8076-359147aefab2.png)